lenovo/thinkpad/x260/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  imports = [
+    ../.
+    ../acpi_call.nix
+    ../../../common/cpu/intel
+  ];
+
+  # See https://linrunner.de/en/tlp/docs/tlp-faq.html#battery https://wiki.archlinux.org/index.php/TLP#Btrfs
+  services.tlp.extraConfig = ''
+    SATA_LINKPWR_ON_BAT=max_performance
+
+    START_CHARGE_THRESH_BAT0=75
+    STOP_CHARGE_THRESH_BAT0=80
+    START_CHARGE_THRESH_BAT1=75
+    STOP_CHARGE_THRESH_BAT1=80
```

Comment:
Is that specific to this model? Is the rationale to save up battery charge cycles?

Reply:
See [here](https://linrunner.de/en/tlp/docs/tlp-faq.html#battery)
>Factory settings for ThinkPad battery thresholds are as follows: when plugged in the battery starts charging at 96%, and stops at 100%. These settings are optimized for maximum runtime, but having a battery hold a lot of power will decrease its capacity over the years. To alleviate this problem, the start/stop charge thresholds can be adjusted – at the cost of a more or less reduced battery runtime.

These values are rather conservative defaults from the TLP manual.